src/rawchannel.cc

```diff
@@ -417,7 +493,7 @@ class RawChannel : public Nan::ObjectWrap
 
     can_err_mask_t err_mask = (can_err_mask_t) info[0]->ToUint32(context).ToLocalChecked()->Value();
 
-    setsockopt(hw->m_SocketFd, SOL_CAN_RAW, CAN_RAW_ERR_FILTER, &err_mask, sizeof(err_mask));
+    setsockopt(hw->m_SocketFd, SOL_CAN_RAW, CAN_RAW_FILTER, NULL, 0);  
```

Comment:
this change breaks error filtering

Reply:
Code restaured, no need to modify this section (test done)